checkpoint.go

```diff
@@ -118,7 +118,7 @@ func setPageServer(context *cli.Context, options *libcontainer.CriuOpts) {
 		if err != nil || address == "" || port == "" {
 			fatal(errors.New("Use --page-server ADDRESS:PORT to specify page server"))
 		}
-		portInt, err := strconv.Atoi(port)
+		portInt, err := strconv.ParseInt(port, 10, 32)
```

Comment:
(I can't comment on the commit message, so I'm commenting here instead).

It would be nice to describe what the linter warning was, and the nature of the fix, in the commit message.

The better commit subject is perhaps something like `checkpoint.go: use ParseInt`

Reply:
@SaschaSchwarze0 ^^^^